editor/test/actions.js

```diff
@@ -1,70 +1,474 @@
+				type: 'TOGGLE_SIDEBAR_PANEL',
+				panel: 'panelName',
+			} );
+		} );
+	} );
+
+	describe( 'createNotice', () => {
+		const status = 'status';
+		const content = <p>element</p>;
+		it( 'should return CREATE_NOTICE action when options is empty', () => {
+			const result = actions.createNotice( status, content );
+			expect( result.type ).toEqual( 'CREATE_NOTICE' );
+			expect( result.notice.status ).toEqual( 'status' );
+			expect( result.notice.content ).toEqual( <p>element</p> );
+			expect( result.notice.isDismissible ).toEqual( true );
+			expect( result.notice.id ).toBeDefined();
+		} );
+		it( 'should return CREATE_NOTICE action when options is desined', () => {
+			const options = {
+				id: 'id',
+				isDismissible: false,
+			};
+			const result = actions.createNotice( status, content, options );
+			expect( result ).toEqual( {
+				type: 'CREATE_NOTICE',
+				notice: {
+					id: 'id',
+					status: 'status',
+					content: <p>element</p>,
+					isDismissible: false,
+				},
+			} );
+		} );
+	} );
+	describe( 'createSuccessNotice', () => {
+		it( 'should return CREATE_NOTICE action', () => {
+			const result = actions.createSuccessNotice( <p>element</p> );
+			expect( result.type ).toEqual( 'CREATE_NOTICE' );
+			expect( result.notice.status ).toEqual( 'success' );
+			expect( result.notice.content ).toEqual( <p>element</p> );
+			expect( result.notice.isDismissible ).toEqual( true );
+			expect( result.notice.id ).toBeDefined();
+		} );
+	} );
+	describe( 'createInfoNotice', () => {
+		it( 'should return CREATE_NOTICE action', () => {
+			const result = actions.createInfoNotice( <p>element</p> );
+			expect( result.type ).toEqual( 'CREATE_NOTICE' );
+			expect( result.notice.status ).toEqual( 'info' );
+			expect( result.notice.content ).toEqual( <p>element</p> );
+			expect( result.notice.isDismissible ).toEqual( true );
+			expect( result.notice.id ).toBeDefined();
+		} );
+	} );
+	describe( 'createErrorNotice', () => {
+		it( 'should return CREATE_NOTICE action', () => {
+			const result = actions.createErrorNotice( <p>element</p> );
+			expect( result.type ).toEqual( 'CREATE_NOTICE' );
+			expect( result.notice.status ).toEqual( 'error' );
+			expect( result.notice.content ).toEqual( <p>element</p> );
+			expect( result.notice.isDismissible ).toEqual( true );
+			expect( result.notice.id ).toBeDefined();
+		} );
+	} );
+	describe( 'createWarningNotice', () => {
+		it( 'should return CREATE_NOTICE action', () => {
+			const result = actions.createWarningNotice( <p>element</p> );
+			expect( result.type ).toEqual( 'CREATE_NOTICE' );
+			expect( result.notice.status ).toEqual( 'warning' );
+			expect( result.notice.content ).toEqual( <p>element</p> );
+			expect( result.notice.isDismissible ).toEqual( true );
+			expect( result.notice.id ).toBeDefined();
+		} );
+	} );
+	describe( 'removeNotice', () => {
+		it( 'should return REMOVE_NOTICE actions', () => {
+			const id = 'id';
```

Comment:
`id` is used only once. It should be either inlined or reused in the `toEqual` call. The same applies to `location` and `feature` in the 2 next tests.